ckanext/zarr/helpers.py

```diff
@@ -122,15 +122,12 @@ def month_formatter(month):
     return datetime.strptime(month, "%Y-%m").strftime("%b %Y")
 
 
-def multiple_select_formater(value):
-    if isinstance(value, str):
-        cleaned_value = value.strip().strip("[]'{}")
-        pattern = r"([a-zA-Z0-9_-]+(?:,[a-zA-Z0-9_-]+)*)"
-        match = re.match(pattern, cleaned_value)
-        if match:
-            inside_content = match.group(0)
-            elements = [element.strip() for element in inside_content.split(',')]
-            return elements
-        else:
-            return ''
-
+def multiple_select_formatter(value):
+    from markupsafe import Markup
```

Comment:
General style guidelines for python is to put all imports at the start of the file. Can we move these out please?